packages/components/buttons/accessible-button/src/accessible-button.tsx

```diff
@@ -91,7 +91,7 @@ const defaultProps: Pick<
 };
 
 const AccessibleButton = forwardRef<HTMLButtonElement, TAccessibleButtonProps>(
-  (props, ref) => {
+  (props: TAccessibleButtonProps, ref) => {
```

Comment:
So this makes it work? Interesting...

Any idea why the types in the `forwardRef` were not used? 🤔

Reply:
I mean this `forwardRef<HTMLButtonElement, TAccessibleButtonProps>`. This should make the `props` already typed.

Reply:
Yes, you are correct. This is how forwardRef is typed but I guess for some reason, where we are using [reactDocgen](https://github.com/commercetools/ui-kit/blob/84e45e0619e96b6c585c8f1f048a0025400881e6/generators/readme/src/index.ts#L332) , it does not parse the complete prop attributes using that syntax.